services/core-api/app/api/parties/party_appt/resources/mine_party_appt_resource.py

```diff
@@ -54,11 +57,18 @@ def get(self, mine_party_appt_guid=None):
                 raise NotFound('Mine Party Appointment not found')
             result = mpa.json(relationships=relationships)
         else:
+            current_app.logger.info(request.args)
```

Comment:
Again, we should probably remove the logging used for debug unless there was another purpose.